### PR TITLE
updated ExecutionRequested schema and regenerated libraries

### PR DESCRIPTION
**Pull request by nelsonkopliku**
Fixes a small typo, as here https://github.com/trento-project/wanda/pull/15

### CODE REVIEW

**Review by arbulu89 — APPROVED** [review]

File: go/pkg/gen/entities/execution_requested_v1.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ import (
 // ExecutionRequestedV1 
 type ExecutionRequestedV1 struct {
   ExecutionId string `json:"execution_id"`
-  GroupId string `json:"group_id,omitempty"`
```
Comment (nelsonkopliku) [inline go/pkg/gen/entities/execution_requested_v1.go:21]: This is interesting. How is it possible that it was already `GroupId` :thinking:
Only the `omitempty` was removed :eyes: 
@CDimonaco what do you think?

Comment (arbulu89) [reply go/pkg/gen/entities/execution_requested_v1.go:21]: Well, the field in the schema was already `group_id`, the error was in the `required`. That's why it only changed the `omitempty`, as now it understood as required (so it cannot be omitted)

Comment (nelsonkopliku) [reply go/pkg/gen/entities/execution_requested_v1.go:21]: AHA! thanks!

Comment (nelsonkopliku) [reply go/pkg/gen/entities/execution_requested_v1.go:21]: However there is a different behavior with the elixir counterpart :thinking: 
anyway, maybe it's not worth bothering, at least immediately.

**Pull request merged**